readme.md

```diff
@@ -81,18 +83,25 @@ Your Rails database has two tables. `students` has the columns `id` and `name`,
 Use ActiveRecord to create a new `pbj` sandwich and make it belong to the student named Geraldo.
 
 ```rb
-# Your answer...
+geradlo = Student.create(name: "Geraldo")
+Sandwich.create(type: "pbj", student_id: "Geraldo")
+
+#I don't know geradlo's id number
 
 ```
 
 ## Question 7
 
-Using Ruby, instantiate an array called `fruits` that contains `apple`, `banana`, and `orange`.
+Using Ruby, instantiate an array called `fruits` that contains `pineapple`, `banana`, and `kiwi`.
 
 Then, use an enumerator to print to the console the sentence "I'd like to eat a [fruit]" once for each fruit.
 
 ```rb
-# Your answer...
+fruits = ["pineapple", "banana", "kiwi"]
+fruits.each do |fruit|
+  puts "I'd like to eat a #{fruit}"
+end
```

Comment:
:+1: 
hahah, I like the updated fruits, in addition to improved grammar, they're more interesting / delicious